front-end/src/domains/friends/components/friends-list/friend-list-item.component.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC, useMemo } from "react";
+
+export interface FriendListItemProps {
```

Comment:
Hey! This is an interface, when it could be a type instead. Try writing it this way:

```
export type BetterType = {value: "weeee";}